packages/manager/modules/vrack/src/dashboard/vrack.controller.js

```diff
@@ -328,7 +336,9 @@ export default class VrackMoveDialogCtrl {
           allServices.dedicatedServerInterface = [];
         }
 
-        return allServices;
+        return pickBy(allServices, (_, serviceType) => {
+          return this.features.isFeatureAvailable(FEATURE_NAMES[serviceType]);
+        });
```

Comment:
related to this:
https://github.com/ovh/manager/pull/5385/files#r684256052

Reply:
This is done